docs/offline-environment.md

```diff
@@ -103,7 +103,9 @@ If you use the settings like the one above, you'll need to define in your invent
   can store them anywhere as long as it's accessible by kubespray. It's recommended to use `*_version` in the path so
   that you don't need to modify this setting everytime kubespray upgrades one of these components.
 * `yum_repo`/`debian_repo`/`ubuntu_repo`: OS package repository depending on your OS, should point to your internal
-  repository. Adjust the path accordingly.
+  repository. Adjust the path accordingly. Used only for Docker/Containerd packages (if needed); other packages might
```

Comment:
Is that still correct ? I mean, are we not getting containerd from upstream rather than distro packages ?

Reply:
TBH, I've not checked all cases. But under Ubuntu, you are correct, at least under Ubuntu 20/22 we do not install it from apt repos. But I was not sure if it is true in all cases :P This is why I added the "(if needed)" part. :D

Reply:
I think that's the case, but it's probably worth checking

Reply:
No, it is no longer used. If you agree, I'll remove this section from this doc completely. The variable is still there, but it is only used to remove the repository from old installs.